app/services/charging-module/create-bill-run.service.js

```diff
@@ -5,11 +5,12 @@
  * @module ChargingModuleCreateBillRunService
  */
 
-const ChargeModuleTokenService = require('./charge-module-token.service.js')
-const RegionModel = require('../models/water/region.model.js')
-const RequestLib = require('../lib/request.lib.js')
+const RegionModel = require('../../models/water/region.model.js')
+const RequestLib = require('../../lib/request.lib.js')
 
-const servicesConfig = require('../../config/services.config.js')
+const servicesConfig = require('../../../config/services.config.js')
+
+const ChargingModuleTokenService = require('./token.service.js')
```

Comment:
With the dotenv fix can this now go back with the rest of the module `require()` statements?

Reply:
Yes good catch -- reverted this now